generator/Generator.sc

```diff
@@ -1131,13 +1131,18 @@ def generateMainClasses(): Unit = {
                   case 2 => BiFunctionType
                   case _ => s"Function$i"
                 }
+                val accessModifier = i match {
+                  case 1 => "transient final"
+                  case 2 => "transient final"
+                  case _ => "final"
```

Comment:
Can't it be `transient final` everywhere?

Reply:
Yes we can keep `transient final` for all `Case`, Although `Case3` onwards uses `io.vavr.Function` which is already serializable, hence there is no serializable warning for `Case3` onwards.

Let me know if I should keep `transient final` for all `Case`.

Reply:
Ah, thanks for the clarification - this is ok